### ISSUE TITLE
422: Unprocessable Entity

### ISSUE THREAD

**Issue by albino1** [theotherp/nzbhydra#96]
I seem to be unable to add anything to SABnzbd right now, and I'm not sure what went wrong.  The errors aren't outputted to the console, they're only visible in the GUI.  The errors I'm getting are:

```
An error occured :
422: UNPROCESSABLE ENTITY

[object Object]
```

And:

`An unexpected error occurred while trying to contact NZB Hydra or add the NZB`

Here's a screenshot:

![al-nzbhydra-unprocessableentity](https://cloud.githubusercontent.com/assets/12164957/12227048/3880b16c-b7da-11e5-9817-cc0669f136fb.jpg)

Downloader: SABnzbd 0.7.20
NZB access type: Proxy NZBs from indexer
NZB adding type: Send link

**Comment by theotherp** [comment]
Which indexer? I cannot reproduce this.

**Comment by theotherp** [comment]
Please open the developer tools in your browser (F12), go to the connections tab, try agan and see what URL is called. Remove any sensitvie data and post it.

**Comment by albino1** [comment]
In trying to figure out how to get you that data I realized it wasn't happening in Chrome, so I restarted Firefox on my laptop and the error went away.  It was still happening on my PC, but when I restarted Firefox on the PC it went away as well.  Maybe there was something cached from before my latest update, but it was awfully persistent through numerous searches.  Sorry to have wasted your time.

In case it helps on any level, this is the only thing I managed to screenshot before I restarted Firefox:

![al-nzbhydra-unprocessableentity-firefox](https://cloud.githubusercontent.com/assets/12164957/12229842/44d945f2-b800-11e5-99fd-da55f2a2722e.jpg)
